### ISSUE TITLE
Player model stuck inside invisible block

### ISSUE THREAD

**Issue by BigBadKangaroo** [tsunamayo/Starship-EVO#5965]
Hi there,

after building in my new world for like 30 minutes, my player model got stuck inside invisbile deleted blocks.
I was unable to delete these invisible blocks.
After a game restart, the invisible blocks disapeared and I could build again without getting stuck.

https://github.com/user-attachments/assets/bf8da66b-063d-48fb-ae21-dc60f082dcee

[30082024 2.zip](https://github.com/user-attachments/files/16824782/30082024.2.zip)
[Player.log](https://github.com/user-attachments/files/16824783/Player.log)

**Comment by tsunamayo** [comment]
@BigBadKangaroo Him I can clearly see an issue with the physics engine, but I dont really know what triggered it... Do you have any recollection of anything specific, beside deleting that block?

**Comment by BigBadKangaroo** [comment]
@tsunamayo I was just building my ship. I use quite a lot of Copy&Paste / Box-Delete paired with the usual x/y symetry axes while building. I remember that I was building some turrets on a different entity and copying them to the main BP. So I was building and deleting some CE's that way.
Maybe this is what caused the physics engine to bug out? Too much C&P, Box-Delete and then switching between entities to build/delete CE's? 

I can't reproduce it relaible either. It just happens when building for a while - however, sometimes it doesn't happen at all.

**Comment by tsunamayo** [comment]
@BigBadKangaroo Is it something that happens since last patch? I did some change in the physics engine.

**Comment by BigBadKangaroo** [comment]
@tsunamayo yes, this started to happen after the update. This has not been a problem at all before the update.

**Comment by tsunamayo** [comment]
@BigBadKangaroo Hi, tell me if it is better now, I think it should be fixed. Cheers
